uint8 onoffdc
uint8 axis
uint8 bank
uint16 msecs_to
---
int8 confirm